src/core/hub/Holdings.sol

```diff
@@ -129,6 +129,18 @@ contract Holdings is Auth, IHoldings {
         _callOnSync(poolId, scId, centrifugeId, snapshot_);
     }
 
+    /// @inheritdoc IHoldings
+    function snapshotTransferShares(
```

Comment:
Not blocked. 

Should we unify names this and `checkSnapshot` to indicate that both does more or less the same but with different callbacks?

Reply:
But with onSyncSnapshot or onTransferSnapshot it seems the methods are called as hook, when they are called the hook (the other way arround)